l2-contracts/contracts/bridge/L2Weth.sol

```diff
@@ -51,26 +54,31 @@ contract L2Weth is ERC20PermitUpgradeable, IL2Weth, IL2StandardToken {
     /// @notice This function is used to integrate the previously deployed WETH token with the bridge.
     /// @param _l2Bridge Address of the L2 bridge
     /// @param _l1Address Address of the L1 token that can be deposited to mint this L2 WETH.
-    function initializeV2(address _l2Bridge, address _l1Address) external reinitializer(2) {
+    function initializeV2(address _l2Bridge, address _l1Address, bool _isEthBaseToken) external reinitializer(2) {
```

Comment:
Just in case, the Weth token has been deployed on Era (and so initializeV2 makes sense), but it has not been deployed on other (future) chains and so I am not sure if it will work out fine. 

I think that probably the logic of initializeV2 should duplicate both the old logic and the new one so that it works everywhere (and maybe remove the `initializer` function altogether)

Reply:
True, good catch, thanks.